### ISSUE TITLE
{Question} Functional?

### ISSUE THREAD

**Issue by felixekman** [projectgus/novepad#1]
Did you ever get this to work with touch? Can't find a definitive statement about it.

**Comment by projectgus** [comment]
No. :( The thread on kosagi forums has some details from back when I was working on it:
http://www.kosagi.com/forums/viewtopic.php?id=58

Summary:
- The Focaltech touch chip doesn't appear to work at all without firmware, which I didn't have. Company & the distributors I found never answered my emails (understandable), and I couldn't find anything suitable in random android file drops, etc.
- I couldn't find any other captouch chips that seemed suitable (in particular, there arelots of touch channels on the ipad3 touch screen digitisers and most chips can't deal with that many).

It's been a couple of years though, so something newer and more suitable might exist now. Another thing to possibly try is to find another model capacitive touch digitiser of the same size, maybe one from a 10 inch Android tablet where the digitiser chip is integrated into the screen unit and it just speaks i2c or something...
